.github/workflows/black.yml

```
@@ -0,0 +1,10 @@
+name: Lint
+
+on: [push, pull_request]
```

`push` will cover PRs as well.
```suggestion
on: push
```